iOSMoviesApp/CustomTableViewCells/CastCollectionViewCell/CastCollectionViewCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  CastCollectionViewCell.swift
+//  iOSMoviesApp
+//
+//  Created by Yash Agrawal on 11/07/25.
+//
+
+import UIKit
+
+
+protocol CastCollectionViewCellDelegate: AnyObject {
+    func updateProfilePic(with imageData : Data, cacheKey: NSString)
+    func updateProfilePicFromCache(with image: UIImage)
+    func updateProfilePicFromPlaceholder(named: String)
+}
+
+class CastCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet weak var realName: UILabel!
```

Comment:
```suggestion
    @IBOutlet weak var realName: UILabel! {
     didSet {
             realName.numberOfLines = 0
             realName.lineBreakMode = .byWordWrapping
    }
```
You can also set attribites like this as well.

Reply:
Made an UILabel Extension instead